ansible/group_vars/wakuv2-test.yml

```diff
@@ -9,6 +9,7 @@ bootstrap__active_extra_users:
 # Tag dependent on fleet: test, prod
 nim_waku_cont_tag: 'deploy-v2-{{ stage }}'
 nim_waku_cont_name: 'nim-waku-v2'
+nim_waku_protocols_available: ['relay', 'store', 'filter', 'swap', 'lightpush', 'rln-relay']
```

Comment:
@jakubgs I added rln-relay to the set of enabled protocols, though, it seems it does not get mounted on the test fleets. Figured that it might be because it is not included in the set of available protocols considering this line of the docker-compose template https://github.com/status-im/infra-role-nim-waku/blob/4c71b953d326f1f12982174eb42020faf07b865f/templates/docker-compose.yml.j2#L37

Reply:
I also think a more correct way of doing this is to augment the default set of available protocols as defined in https://github.com/status-im/infra-role-nim-waku/blob/1e5367c86a906d9b08c0d492f5d70a85d790ae37/defaults/main.yml#L15 with `rln-relay` (instead of overriding it entirely), what do you think? @jakubgs

Reply:
Yes, it should be set in the defaults.

Reply:
Also, there is no automation that deploys changes in Ansible roles. There never was.